pages/osx/whatis.md

```diff
@@ -0,0 +1,12 @@
+# whatis
+
+> Tool that searches a set of database files containing short descriptions of system commands for keywords.
+> More information: <http://www.linfo.org/whatis.html>.
+
+- Search for informations about keyword:
```

Comment:
`informations` is grammatically incorrect. Let's also change to "about _a_ keyword"

Reply:
thanks @agnivade I didn't know `information` is non-countable word. I will make another PR for the fix.